### ISSUE TITLE
AnchorPlugins.com

### ISSUE THREAD

**Issue by daviddarnes** [anchorcms/anchor-plugins#1]
**First issue to kick things off on this repo, the site for it.**

I realise none of the plugins work is stable enough to start work on a directory site, but we already do have a few assets in hand. For one @CorstianBoerman has the domain http://anchorplugins.com, which I did ask for it to be directed to https://anchorthemes.com/plugins. However it seems to be timing out, so I think it's be neglected.

So how do people want to approach plugins for Anchor? I was hoping to follow a similar pattern to Themes, which would eliminate the use for this repo. We could also use this as a registry, any repo approved and listed in this repo is listed on the plugins site and is available to download to Anchor CMS. We could also reflect this technique to themes.
### To do:
- [x] Get domain
- [ ] Setup domain
- [x] Get Twitter account
- [ ] Create site
- [x] Get a list of possible plugins
- [x] Create plugins repo
- [ ] Finish plugins work on Anchor CMS
- [ ] Setup a verification process for plugins

Thoughts?

**Comment by noahbuscher** [comment]
Personally, I like the idea of using this repo as a registry, with a website that pulls the plugins and allows users to download them. I think that's what ST's Package Manager did (or does).

**Comment by daviddarnes** [comment]
@noahbuscher thanks for the feedback! Glad you like the idea. We'd get the best of both worlds if this was a registry of people's plugin repos, maybe using versioning to make sure they're stable.

I guess it could just be a json or yml list of repos with releases, any updates would need to combined into a release and updated in the registry.

**Comment by corstian** [comment]
I'm on the domain timeout when I'm done with work today.

**Comment by daviddarnes** [comment]
@CorstianBoerman cool. You have the Twitter account too I guess?

**Comment by robmdev** [comment]
I've already created the ability to feed multiple object types in to custom fields - I can push it in to a plugin if nobody else has already started it.

I wrote a whole new media section for it so in my Summernote WYSIWYG editor, I can access the images and it just calls them by slug. :)

![screen shot 2015-12-15 at 14 30 03](https://cloud.githubusercontent.com/assets/13496654/11813285/76b8de8e-a338-11e5-9413-577d11b42900.png)
![screen shot 2015-12-15 at 14 30 47](https://cloud.githubusercontent.com/assets/13496654/11813288/7918c6a8-a338-11e5-8897-fc0404f7b610.png)

**Comment by daviddarnes** [comment]
@robmdev would be cool to bundle this into a plugin, but we need to work out the best method first :). But you should absolutely push this to it's own repo!

**Comment by robmdev** [comment]
@daviddarnes Absolutely. I will branch off a new one because I made some changes I no longer like. Lol! I'll sort out a new version with the media manager. 

With plugins, would it be viable to look at hooks, much like Slim PHP Framework? It's a very similar code style so could be useful to have a look at that. Cheers!

**Comment by CSCG** [comment]
@daviddarnes is this still open for work?

**Comment by daviddarnes** [comment]
@CSCG it is, but it's gone stale sadly. Do you have a plugin in mind? We need to get the feature added before we setup a page for it
